locales/en/messages.json

```diff
@@ -3710,7 +3710,39 @@
         "message": "Cell Count - for auto Profile switching"
     },
     "pidTuningCellCountHelp": {
-        "message": "Automatically activates the first profile that has a cell count equal to that of the connected battery."
+        "message": "Automatically activates the first profile that has a cell count equal to that of the connected battery.<br><br><ul><li><b>Switch</b>: Always switch to a profile with matching cell count if there is one.</li><li><b>Disable</b>: Disable auto profile switching</li><b>1S-8S</b>: Select the profile cell count to be used for this profile</li></ul>"
+    },
+    "pidTuningCellCountChange": {
+        "message": "Switch",
+        "desciption": "Use this profile if there no profiles matching cell count"
```

Comment:
```suggestion
        "description": "Use this profile if there no profiles matching cell count"
```

Reply:
The description is used to give context to translators. I'm not too sure if this description can help. Is not a good tooltip than a context for the word.

Reply:
Fixed.